### PR TITLE
Fix skipped tests

### PR DESCRIPTION
**Pull request by IKarbowiak**
Fix skipped tests:
- Fix translation query for Sale
- Fix subscription webhooks for Sale

Partially covers https://github.com/saleor/saleor/issues/13543 (partially it was covered in the PR that were updating mutations) 

# Impact

- [ ] New migrations
- [ ] New/Updated API fields or mutations
- [ ] Deprecated API fields or mutations
- [ ] Removed API types, fields, or mutations

# Docs

- [ ] Link to documentation:

# Pull Request Checklist

- [ ] Privileged queries and mutations are either absent or guarded by proper permission checks
- [ ] Database queries are optimized and the number of queries is constant
- [ ] Database migrations are either absent or optimized for zero downtime
- [ ] The changes are covered by test cases

### CODE REVIEW

File: saleor/graphql/webhook/tests/mutations/test_webhook_dry_run.py [hunk 1]
```diff
@@ -605,7 +605,6 @@ def async_subscription_webhooks_with_root_objects(
     }
 
 
-# TODO will be fixed in PR refactoring the mutation
 @pytest.mark.skip
```
Comment (zedzior) [inline saleor/graphql/webhook/tests/mutations/test_webhook_dry_run.py:608]: Removed comment, but test is still skipped. Is it intended?

Comment (IKarbowiak) [reply saleor/graphql/webhook/tests/mutations/test_webhook_dry_run.py:608]: Nope. Thanks!

File: saleor/graphql/webhook/mutations/webhook_dry_run.py [hunk 2]
```diff
@@ -102,10 +104,18 @@ def validate_input(cls, info, **data):
...
 
         return event_type, object, query
 
+    @classmethod
+    def get_instance(cls, info: ResolveInfo, object_id):
+        type, _id = from_global_id_or_error(object_id, raise_error=False)
+        if type == "Sale":
+            object_id = cls.get_global_id_or_error(object_id, "Sale")
+            return discount_models.Promotion.objects.get(old_sale_id=object_id)
+        return cls.get_node_or_error(info, object_id, field="objectId")
```
Comment (zedzior) [inline saleor/graphql/webhook/mutations/webhook_dry_run.py:117]: We need to do the same for `webhookTrigger` mutation.

**Review by zedzior — CHANGES_REQUESTED** [review]

**Review by fowczarek — APPROVED** [review]

File: saleor/graphql/webhook/mutations/webhook_dry_run.py (hunk 2 above)
Comment (IKarbowiak) [reply saleor/graphql/webhook/mutations/webhook_dry_run.py:117]: Done

**Review by zedzior — APPROVED** [review]

**Review by tomaszszymanski129 — APPROVED** [review]

**Pull request merged**